build.gradle

```diff
@@ -81,7 +82,7 @@ ext {
     // Compatible with Kafka version:
     // https://docs.confluent.io/current/installation/versions-interoperability.html
     confluentPlatformVersion = "4.1.4"
-    aivenConnectCommonsVersion = "0.8.2"
```

Comment:
Use snapshot version of `commons-for-apache-kafka-connect`. Will be changed before merging to released commons version.